testdata/goldens/function_with_return_value_custom_template.go

```diff
@@ -0,0 +1,17 @@
+package testdata
+
+import "testing"
+
+func TestFoo4(t *testing.T) {
```

Comment:
Where is this template output different than the default one?

Reply:
it renames `tests` to `testCases`

Reply:
Aw, I see. Thanks!